components/date-time/date.js

```diff
@@ -2,7 +2,11 @@
  * External dependencies
  */
 import ReactDatePicker from 'react-datepicker';
-import moment from 'moment';
+
+/**
+ * WordPress dependencies
+ */
+import { moment } from '@wordpress/date';
```

Comment:
The usage of the regular `moment` was intentional here. The idea is that we don't need any locale set, we just use moment as an utility function without global locale. (this avoids a dependency between `components` and `date` which I think would be nice to keep independent.)

Reply:
Reverting those changes :+1:
I wasn't aware of this. It still would be useful to wrap those `moment` calls with some higher level API to make it possible to replace `moment` with something else as soon as there is good alternative for it :)